dissect/target/filesystems/zip.py

```diff
@@ -71,9 +72,9 @@ def detect(fh: BinaryIO) -> bool:
         finally:
             fh.seek(offset)
 
-    def get(self, path: str) -> ZipFilesystemEntry:
+    def get(self, path: str, relentry: FilesystemEntry = None) -> ZipFilesystemEntry:
         """Returns a ZipFilesystemEntry object corresponding to the given path."""
-        return self._fs.get(path)
+        return self._fs.get(path, relentry)
```

Comment:
```suggestion
        return self._fs.get(path, relentry=relentry)
```
Otherwise we may get in trouble if the function signature changes.